lib/logflare/sql.ex

```diff
@@ -1390,6 +1414,12 @@ defmodule Logflare.Sql do
   defp is_identifier?(identifier),
     do: is_map_key(identifier, "CompoundIdentifier") or is_map_key(identifier, "Identifier")
 
+  defp is_numeric_value?(%{"Value" => %{"Number" => _}}), do: true
+  defp is_numeric_value?(_), do: false
+  defp is_json_access?(%{"Nested" => %{"JsonAccess" => _}}), do: true
+  defp is_json_access?(%{"JsonAccess" => _}), do: true
+  defp is_json_access?(_), do: false
```

Comment:
By [Elixir naming convention](https://hexdocs.pm/elixir/naming-conventions.html#trailing-question-mark-foo) it should be `json_access?` and `numeric_value?`. `is_*` prefix (with no question mark at the end) is meant for guards.